pkg/ccl/changefeedccl/sink_webhook_v2.go

```diff
@@ -334,6 +404,7 @@ func (jb *webhookJSONBuffer) Close() (SinkPayload, error) {
 		buffer.Write(msg)
 	}
 	buffer.WriteString(suffix)
+
```

Comment:
nit: rm

Reply:
done

Reply:
It appears to still be here.

Reply:
It's fixed [here](https://github.com/cockroachdb/cockroach/pull/138872/commits/48667618edc92e032ee9965a0a00e2ed50877390#diff-8b79b659227dd56b0190adcc52faebe0116f8abed96e0284359e810b21f361b9L407)

Reply:
btw should I squash commits each time?

Reply:
You'll need to squash them all into one commit before merging. You can either continuously update the same commit, or squash after the review back-and-forth, up to you.